### PR TITLE
Update 2020-11-21-debenedetto20a.md

### PR DESCRIPTION
**Pull request by davidweichiang**
- First author's name was incorrectly lowercased
- Removed LaTeX command from abstract

### CODE REVIEW

**Review by lawrennd — APPROVED** [review]
Thanks David!

File: _posts/2020-11-21-debenedetto20a.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ url: http://proceedings.mlr.press/v119/debenedetto20a.html
 abstract: Unordered, variable-sized inputs arise in many settings across multiple
   fields. The ability for set- and multiset-oriented neural networks to handle this
   type of input has been the focus of much work in recent years. We propose to represent
-  multisets using complex-weighted \emph{multiset automata} and show how the multiset
+  multisets using complex-weighted multiset automata and show how the multiset
```
Comment (lawrennd) [inline _posts/2020-11-21-debenedetto20a.md]: ``
```suggestion
  multisets using complex-weighted <i>multiset automata</i> and show how the multiset
```

File: _posts/2020-11-21-debenedetto20a.md [hunk 2]
```diff
@@ -25,10 +25,10 @@ lastpage: 2420
 page: 2412-2420
 order: 2412
 cycles: false
-bibtex_author: Debenedetto, Justin and Chiang, David
+bibtex_author: DeBenedetto, Justin and Chiang, David
```
Comment (lawrennd) [inline _posts/2020-11-21-debenedetto20a.md]:
```suggestion
bibtex_author: {DeBenedetto}, Justin and Chiang, David
```

**Review by lawrennd — APPROVED** [review]
Minor fixes

**Comment by lawrennd** [comment]
Thanks David!

**Comment by davidweichiang** [comment]
Thank you, but now the paper seems to have vanished from the proceedings! It's not on the table of contents and the link (http://proceedings.mlr.press/v119/debenedetto20a.html) is a 404.

**Comment by lawrennd** [comment]
Whoops, thanks for checking David. I needed quotes around the bibtex author names when curly brackets are in place. Fixed now!

**Pull request merged**